Item.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+class Item:
+
+    def parseDue(self, due):
+        dueFormatted = "No Due Date"
+        overdue = False
+        if due:
+            dueDate = datetime.datetime.strptime(due, '%Y-%m-%dT%H:%M:%S.%f%z')
+            dueFormatted = dueDate.strftime("%d/%m/%Y")
+            overdue = self.status != 'Done' and dueDate.date() <= datetime.date.today()
+        return dueFormatted, overdue
```

Comment:
Unimportant suggestion for an improvement here:

Would it be neater to instead save dueDate as a property on Item and have functions `getFormattedDueDate()` and `isOverdue()` ?

Plus it would be impossible to get inconsistencies if you modify Item for whatever reason. And it's easier to get the parsed datetime object to perform any other logic.

Reply:
Good idea. Fixed in [d69bfbb](https://github.com/rysharprules/DevOps-Course-Starter/pull/2/commits/d69bfbbe88182ea5c84bc5e4306e9a9d4d10fc28)